src/commands/decompress.rs

```diff
@@ -31,6 +31,7 @@ pub struct DecompressOptions<'a> {
     pub formats: Vec<Extension>,
     pub output_dir: &'a Path,
     pub output_file_path: PathBuf,
+    pub is_output_dir_explicit: bool,
```

Comment:
Can you call this `is_output_dir_provided` or `is_dir_flag_present` or `has_dir_flag`